File: src/sdk-reference/java/1/auth/validate-my-credentials/snippets/validate-my-credentials.java
try {
    kuzzle.getAuth().login("local", "{\"username\":\"foo\",\"password\":\"bar\"}");
    kuzzle.getAuth().validateMyCredentials("local", "{\"username\":\"foo\",\"password\":\"bar\"}");
    System.out.println("Success");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}

File: src/sdk/cpp/1/controllers/auth/validate-my-credentials/snippets/validate-my-credentials.cpp
try {
  kuzzle->auth->login("local", R"({"username":"foo","password":"bar"})");
  bool valid = kuzzle->auth->validateMyCredentials("local", R"({"username":"foo","password":"bar"})");

  if (valid) {
    std::cout << "Credentials are valid" << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
